src/components/LoginDialog.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <q-card style="min-width: 25%">
+    <q-card-section>
+      <div class="text-h6">
+        Log in
+      </div>
+    </q-card-section>
+    <form @submit.prevent="onSubmit">
+      <q-card-section>
+        <q-input
+          v-model="username"
+          dense
+          outlined
+          placeholder="Username"
+          stack-label="Stack Label"
+        />
+      </q-card-section>
+      <q-card-section>
+        <q-input
+          v-model="password"
```

Comment:
you can add `type="password"` to hide the user's input as dots

Reply:
Thanks! Will do that right now